code/__DEFINES/_helpers.dm

```diff
@@ -17,10 +17,14 @@
 /// Until a condition is true, sleep
 #define UNTIL(X) while(!(X)) stoplag()
 
+#ifdef EXPERIMENT_515_DONT_CACHE_REF
+// REMOVE this and replace with `ref` when removing EXPERIMENT_515_DONT_CACHE_REF
+#define text_ref(datum) ref(datum)
```

Comment:
text_ref() should stick around even after 515 and 516.

Once we put a define/macro wrapper around some byond function for whatever reason, its better to leave it and give us more power to undercut it in the future then removing that ability.

also REF() should be renamed `html_ref` or `html_text_ref` since its only valid for use in html strings or ui_datas. 

text_ref should at most be renamed to raw_ref.